Modules/III-k6-Intermediate/08-Setting-load-profiles-with-executors/Constant-Arrival-Rate-Exercises.md

```diff
@@ -47,92 +49,111 @@ k6 run test.js
+k6_workshop ✓ [======================================] 00/25 VUs  30s  50.00 iters/s
+
+     data_received..................: 1.2 MB 40 kB/s
+     data_sent......................: 174 kB 5.7 kB/s
+     http_req_blocked...............: avg=4.26ms   min=1µs      med=4µs      max=332.88ms p(90)=8µs      p(95)=14µs    
+     http_req_connecting............: avg=2.05ms   min=0s       med=0s       max=167.98ms p(90)=0s       p(95)=0s      
+     http_req_duration..............: avg=136.87ms min=101.78ms med=114.56ms max=862.77ms p(90)=146.68ms p(95)=184.36ms
+       { expected_response:true }...: avg=136.87ms min=101.78ms med=114.56ms max=862.77ms p(90)=146.68ms p(95)=184.36ms
+     http_req_failed................: 0.00%  ✓ 0         ✗ 1501
+     http_req_receiving.............: avg=46.57µs  min=11µs     med=39µs     max=503µs    p(90)=82µs     p(95)=96µs    
+     http_req_sending...............: avg=17.21µs  min=5µs      med=15µs     max=151µs    p(90)=28µs     p(95)=30µs    
+     http_req_tls_handshaking.......: avg=2.19ms   min=0s       med=0s       max=203.88ms p(90)=0s       p(95)=0s      
+     http_req_waiting...............: avg=136.81ms min=101.73ms med=114.5ms  max=862.74ms p(90)=146.57ms p(95)=184.31ms
+     http_reqs......................: 1501   49.213783/s
+     iteration_duration.............: avg=141.27ms min=101.88ms med=114.83ms max=862.9ms  p(90)=150.38ms p(95)=407.84ms
+     iterations.....................: 1501   49.213783/s
+     vus............................: 25     min=25      max=25
+     vus_max........................: 25     min=25      max=25
 ```
-> :point_up: Feel free to ignore the entries about `VU #0`...it's a bit of a special case.
 
-Reviewing the output now, we see that the desired rate was more closely achieved at 9.43 iterations per second (for the overall test), and that k6 ultimately scaled up to 13 VUs to achieve the desired rate. That's autoscaling!
+Reviewing the output now, we see that the desired rate is more closely achieved at 49.21 iterations per second (for the overall test).
+
+You might be tempted to set a lower value for [`preAllocatedVUs` and use `maxVUs` option](https://k6.io/docs/using-k6/scenarios/executors/constant-arrival-rate/) to autoscale the test. `maxVUs` is the maximum number of VUs to allow during the test run, which defaults to `preAllocatedVUs` when not set. However, it's usually best to leave the default value for `maxVUs` and increase the `preAllocatedVUs`. In this way, the VUs are allocated before the test starts and will be used when (and only if) necessary.
 
-> :point_up: By **not** specifying `maxVUs` in our first test, we essentially disabled autoscaling of VUs.
+Allocating VUs in the middle of the test can be costly in terms of CPU resources in the load generator instance, and can skew the tests. Preallocating VUs will allow the test to start with all the VUs available with no need to wait to allocate more when needed in the middle of the test run.
```

Comment:
Also I try to avoid "allows" since it some readers find it paternalizing. It's not really about permission as much as it is about a clear state of affairs.